### DISCUSSION TITLE
airtable automations import [Oct 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4000] [Discord Threads]
> 🙋‍♂️ **Metaversai**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### automations are not imported. only table schema, views & records  
#  
> 💁 **o1lab**  
#### transferring everything from a closed source application is not that easy 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1023985012221161583)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
